ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -110,7 +141,7 @@ sub remount_everything
     while (my $f = <$fstab>) {
     	my $mount = $self->mount_from_entry($f);
     	$mount && $mount ne 'swap' or next;
-    	LC::File::makedir ($mount);
+    	LC::Check::directory ($mount);
```

Comment:
Thanks for making this change. Can you confirm this now works safely under NoAction mode and then add the NoActionSupported attribute?